PokeAlarm/Manager.py

```diff
@@ -999,7 +1004,8 @@ def process_raid(self, raid):
             'dir': get_cardinal_dir([lat, lng], self.__location),
             'quick_move': self.__locale.get_move_name(quick_id),
             'charge_move': self.__locale.get_move_name(charge_id),
-            'form': self.__locale.get_form_name(pkmn_id, raid_pkmn['form_id'])
+            'form': self.__locale.get_form_name(pkmn_id, raid_pkmn['form_id']),
+            'team': self.__locale.get_team_name(team_id)
```

Comment:
Can we change `<team?` to `<team_name>` and then make sure we include `<team_id>` as well? It may be useful for URLs.